getsolus/packages
# Initial inclusion of eopkg4-bin

**Summary**
Includes eopkg4-bin, which is eopkg4 compiled with nuitka. The resulting package doesn't depend on python, which whould make it much more resilient to system python upgrades.

**Test Plan**

- Build `eopkg4-bin` from this PR
- Install it
- Check to see that all eopkg functions work correctly using the command `eopkg4-bin`. The same caveats apply as with `eopkg4`.



**Checklist**
